examples/Tools/TestWebServer/TestWebServer.ino

```diff
@@ -42,7 +42,7 @@ void setup() {
 
   Serial.println("starting,..");
   // connection state
-  boolean connected = true;
+  boolean connected = false;
 
   // Start GSM shield
   // If your SIM has PIN, pass it as a parameter of begin() in quotes
```

Comment:
Here you forget to change the `&` to `&&` :-) btw I'll merge this PR and add the extra commit needed to fix this